### ISSUE TITLE
500 error should have more pretty output

### ISSUE THREAD

**Issue by MichaelMayorov** [sympy/sympy-bot-old#103]
For example http://reviews.sympy.org/report/ (with '/' in the end) gives traceback, that gives good explanation of problem, but I'm not sure that good idea to show this to normal user.

Also, may be split this issue on "500 error should have more pretty output" and "http://reviews.sympy.org/report/ gives 500" ?

**Comment by certik** [comment]
Yes, so far I enabled all these exceptions for debugging purposes. We might just log this at the server and display some nice page to the user instead.

**Comment by MichaelMayorov** [comment]
Ok, but do we need to have pages for 40x errors?

**Comment by certik** [comment]
I don't know. Did you get any 40x error with the web app?
